charts/apisix-dashboard/README.md

```diff
@@ -0,0 +1,57 @@
+# Apache APISIX Dashboard
+
+[APISIX Dashboard](https://github.com/apache/apisix-dashboard/) is designed to make it as easy as possible for users to operate Apache APISIX through a frontend interface.
+
+This chart bootstraps an apisix-dashboard deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+APISIX Dashboard requires Kubernetes version 1.14+.
+
+## Get Repo Info
+
+```console
```

Comment:
console?

Reply:
😂 Oh, this is a mistake, I will update.

Reply:
> console?

console is used to highlight (display to the user) a sequence of shell session commands (with prompts, not just the contents of the prompt script file).

Reply:
get it